app/components/trainees/confirmation/lead_school/view.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Trainees
+  module Confirmation
+    module LeadSchool
+      class View < GovukComponent::Base
+        include SummaryHelper
+
+        attr_accessor :data_model
+
+        def initialize(data_model:)
+          @data_model = data_model
+          @not_provided_copy = I18n.t("components.confirmation.not_provided")
+        end
+
+        def trainee
+          data_model.is_a?(Trainee) ? data_model : data_model.trainee
+        end
+
+        def summary_title
+          t(".summary_title")
+        end
+
+        def lead_school_name
+          return @not_provided_copy if lead_school.name.blank?
+
+          lead_school.name
```

Comment:
You'll need to explicitly fetch the school via the current id on the trainee otherwise this won't work when stashing changes and we need to show new school values.